src/main/java/com/chattriggers/ctjs/mixins/gui/GuiIngameMixin.java

```diff
@@ -204,5 +220,33 @@ private void chattriggers_renderScoreboardTrigger(
     //$$         ci.cancel();
     //$$     }
     //$$ }
+    //$$
+    //$$ // @WrapWithCondition(
```

Comment:
FYI: These methods are commented out since the current version of MixinExtras bundled with Essential only supports void methods.  This hopefully can be uncommented out after the next Essential update.